openedx/core/djangoapps/schedules/config.py

```diff
@@ -3,12 +3,12 @@
 """
 
 from crum import get_current_request
-from edx_toggles.toggles import LegacyWaffleSwitch, LegacyWaffleFlagNamespace, LegacyWaffleSwitchNamespace, WaffleFlag
+from edx_toggles.toggles import LegacyWaffleSwitch, LegacyWaffleSwitchNamespace, WaffleFlag
 
 from lms.djangoapps.experiments.flags import ExperimentWaffleFlag
 from lms.djangoapps.experiments.models import ExperimentData
 
-WAFFLE_FLAG_NAMESPACE = LegacyWaffleFlagNamespace(name='schedules')
+WAFFLE_FLAG_NAMESPACE = 'schedules'
 WAFFLE_SWITCH_NAMESPACE = LegacyWaffleSwitchNamespace(name='schedules')
```

Comment:
you can pass the `WAFFLE_FLAG_NAMESPACE` to `WAFFLE_SWITCH_NAMESPACE = LegacyWaffleSwitchNamespace(name=WAFFLE_FLAG_NAMESPACE)`

Reply:
switches stay as they were, all corresponding changes will be provided in the following commits